src/cli/task.rs

```diff
@@ -313,10 +313,7 @@ pub fn execute(args: Args) -> miette::Result<()> {
                                 .flat_map(|tasks| {
                                     tasks
                                         .into_iter()
-                                        // .filter(|(_, value)| match value {
-                                        //     Task::Execute(process) => Some(true) != process.hidden,
-                                        //     _ => true,
-                                        // })
+                                        .filter(|&(key, _)| !key.as_str().starts_with('_'))
                                         .map(|(key, _)| key)
```

Comment:
I want to use `filter_map` here as this seems to be the correct thing to do. Just need some more time to get it right. 

no spoilers :slightly_smiling_face:

Reply:
I'm not 100% sold on if `filter_map` is better than the `filter` and `map` combination.